pages/about.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>SD Global Conference</title>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="../css/about.css">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.0/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-KyZXEAg3QhqLMpG8r+8fhAXLRk2vvoC2f3B09zVXn8CA5QIVfZOJ3BCsw2P0p/We" crossorigin="anonymous">
+  <link href="//db.onlinewebfonts.com/c/d14035f6b1afeabafbee4abb2ebf0fc6?family=COCOGOOSE" rel="stylesheet"
+    type="text/css" />
+</head>
+
+<body>
+  <div class="top-bar-section">
+    <div class="top-bar-container">
+      <ul class="top-bar">
+        <li>
+          <a href="#">English</a>
+        </li>
+        <li>
+          <a href="#">My Page</a>
+        </li>
+        <li>
+          <a href="#">Logout</a>
+        </li>
+      </ul>
+    </div>
+  </div>
+  <button title="hamburgermenu" type="button" class="hamburger">
+    <span class="bar"></span>
+  </button>
+  <nav class="mobile-nav">
+    <ul>
+      <li><a href="../pages/about.html">About</a></li>
+      <li><a href="#program">Program</a></li>
+      <li><a href="#join">Join</a></li>
+      <li><a href="#sponsor">Sponsor</a></li>
+      <li><a href="#news">News</a></li>
+    </ul>
+  </nav>
+  <nav class="navbar navbar-expand-md navbar-light bg-light">
+    <div class="container-fluid">
+      <a class="navbar-brand" href="../index.html"><img src="../images/logo1.png" alt="logo"></a>
+      <div class="collapse navbar-collapse" id="navbarSupportedContent">
+        <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+          <li class="nav-item">
+            <a class="nav-link active" aria-current="page" href="#">About</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#">Program</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#">Join</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#">Sponsor</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#">News</a>
+          </li>
+        </ul>
+        <button class="btn btn-outline-success" type="submit">CC Campaign</button>
+      </div>
+    </div>
+  </nav>
+
+  <section class="first-section">
+    <div class="first-section-container">
+      <h3 class="second-heading">"Hello! Sharing world"</h3>
+      <h1 class="open-heading">SOFTWARE DEVELOPER GLOBAL SUMMIT 2021</h1>
+    </div>
+
+    <div class="about-paragraph">
```

Comment:
Kindly fix the position of the text.
![Screen Shot 2021-08-12 at 21 50 03](https://user-images.githubusercontent.com/22258274/129298168-95897cd6-d7cf-4025-b722-b490cc7db9e4.png)